Website/Providers/DataProviders/SqlDataProvider/DotNetNuke.Schema.SqlDataProvider

```diff
@@ -19496,7 +19496,7 @@ As
 	ELSE
 		BEGIN
 			-- AssemblyID is not null - Assembly is registered - test for version
-			SET @CurrentVersion  = (SELECT Version 
+			SET @CurrentVersion  = (SELECT Max(Version ) 
```

Comment:
There is an issue with this comparison. Consider this SQL script and see how it identifies the wrong version as maximum.
```
CREATE TABLE #versions (ver nvarchar(100));
INSERT INTO #versions (ver) VALUES (N'2.0.0');
INSERT INTO #versions (ver) VALUES (N'10.0.0');

select * from #versions;
select MAX(ver) from #versions; -- identifies 2.0.0 as the maximum and not 10.0.0

DROP TABLE #versions;
```